diffcp/cone_program.py

```diff
@@ -225,25 +281,44 @@ def solve_and_derivative(A, b, c, cone_dict, warm_start=None, mode='lsqr',
     return x, y, s, D, DT
 
 
-def solve_and_derivative_internal(A, b, c, cone_dict, solve_method=None,
+def solve_only(A, b, c, cone_dict, warm_start=None, mode='lsqr',
```

Comment:
Ok, I think I see your point: the mode keyword only affects how the derivative of the cone program is being computed. Furthermore, since the `solve_only` functionality doesn't compute the derivative, I should remove the keyword from the entire chain of `solve_only` functions, right?

Reply:
I think so? I might be missing something.

Reply:
No, no - I think you are spot on! Especially given the docstring param description "Differentiation mode in ["lsqr", "lsmr", "dense"]." 
So I'll go make that change and test it